src/typing/flow_js.ml

```diff
@@ -3696,6 +3696,15 @@ let rec __flow cx ((l: Type.t), (u: Type.use_t)) trace =
       ) in
       rec_flow_t cx trace (ArrT (reason, f t, List.map f ts), tout)
 
+    | _, TupleMapT (reason, funt, tout) ->
```

Comment:
> the new error is different but fine.

It seems that it wasn't working as exepected anyway (no "(too few arguments)" part)